src/codeu/chat/server/Model.java

```diff
@@ -182,7 +182,7 @@ public void updateMessageLink(Message message) {
 	  // EDIT - Malik Graham
 	  // Updating the next message for the last message
       try {
-    	  ds.updateLastMessage(message);
+    	  ds.updateNextMessage(message);
```

Comment:
I guess the question is, will that exception ever actually happen, since we're running sql that you wrote (as opposed to sql that a user wrote)? Maybe you can do the exception catching at a different level (inside `DerbyStore`), so calling code doesn't have to worry about it?